backend/homework/1/README.md

```
@@ -2,0 +1,45 @@
+# Homework 1
+
+Demo Video
+
+https://video.drift.com/v/ab4ug4NcThM7TYvHG0TMW4AjoNLaT4WPZP7KwSwstdUM/
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)